velox/expression/tests/CastExprTest.cpp

```diff
@@ -765,7 +783,7 @@ TEST_F(CastExprTest, timestampAdjustToTimezone) {
           Timestamp(946713600, 0),
           Timestamp(0, 0),
           Timestamp(946758116, 0),
-          Timestamp(-21600, 0),
+          Timestamp(-50400, 0),
```

Comment:
this expectation was just wrong. Validated with Presto and bash "date" command.

Reply:
I guess what I'm curious is more of whether this PR introduces a behavior change reflected by this test case. If so, we would need to notify customers and possible run their A/B tests.